docs/build.md

```
@@ -55,9 +55,39 @@ Notes:
+make command line.
+
+```bash
+RPM_OUTDIR=/tmp/microshift-rpms
+make rpm-deb RPM_OUTDIR="${RPM_OUTDIR}"
+```
+
+If the conversion completes successfully, the Debian packages are copied to the
+`${RPM_OUTDIR}/deb` directory on the host. The packages from this directory can
+be used to install MicroShift on the supported operating systems.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)